common/src/main/java/org/wso2/carbon/testgrid/common/constants/TestGridConstants.java

```diff
@@ -13,6 +13,5 @@
     public static final String TEST_TYPE_TESTNG = "TESTNG";
 
     public static final String JMTER_SUFFIX = ".jmx";
-
-
+    public static final String TESTNG_SUFFIX = ".jar";
```

Comment:
Since this is only used in `TestNGTestReader.java` class shall we move it there? Also, will it make more sense if we rename the constant name `TESTNG_SUFFIX` to `JAR_EXTENSION`? WDYT?